pom.xml

```diff
@@ -1927,7 +1927,7 @@
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
-                <version>2.3.1</version>
+                <version>2.8.1</version>
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14005 (2.8.1)

Reply:
GeoServer 2.12.0 has no gson jar.

Reply:
accumulo 1.8 is baselined on 2.7

Reply:
Approved: 
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16003